documentation/library_guide/macros.rst

```diff
@@ -98,6 +98,13 @@ Macro                              Description
 ---------------------------------- ------------------------------
 ``ONEDPL_ALLOW_DEFERRED_WAITING``  This macro allows waiting for completion of certain algorithms executed with
                                    device policies to be deferred. (Disabled by default.)
+
+                                   When the macro evaluates to non-zero, a call to a oneDPL algorithm with
+                                   a device policy might return before the computation completes on the device.
+
+                                   .. Warning:: Before accessing data produced or modified by the call, waiting
```

Comment:
There seems to be a line break in the rendered version between "waiting" and "for".

Reply:
Fixed.